package.json

```diff
@@ -25,12 +25,12 @@
     "@changesets/cli": "^2.16.0",
     "@github/prettier-config": "0.0.4",
     "@primer/primitives": "^5.1.0",
-    "eslint": "^8.0.1",
+    "eslint": "^8.42.0",
```

Comment:
This is compatible with version in dotcom, https://github.com/github/github/blob/master/package-lock.json#L16328

And gives us new features like `getScope`

Reply:
Thank you!!! 🎉 you're the best